src/main/resources/openapi/courthouse.yaml

```diff
@@ -114,6 +317,38 @@ components:
         application/json:
           schema:
             $ref: '#/components/schemas/Courthouse'
+
+  responses:
+    ConstraintViolationError:
+      description: Constraint violation
+      content:
+        application/json+problem:
+          schema:
+            $ref: './problem.yaml'
+          example:
+            type: https://zalando.github.io/problem/constraint-violation
+            status: 400
+            title: Constraint Violation
+
+    CourthouseNotFoundError:
+      description: 'A courthouse corresponding to the provided id could not be found'
+      content:
+        application/json+problem:
+          schema:
+            $ref: './problem.yaml'
+          example:
+            type: "COURTHOUSE_102"
```

Comment:
The PR https://github.com/hmcts/darts-api/pull/1009/files#diff-390a4e5f6fb53180c11ff27794f3974eded49372d093c9c05ea1b8e40a9a8b3d seems to suggest this would be COMMON_100